### DISCUSSION TITLE
belongsToMany table and form builder

### DISCUSSION THREAD

**Discussion by arashbahariyeh** [filamentphp/filament#5350] [Help]
hi all,
i have two models named Company and Sector, a company has many sectors and vice versa hence there is a belongsToMany relationship defined in both models and they are linked via a pivot table named Comoany-Sector. how can i display in companies table all the sectors corresponding to that company?
with belongsTo relationship it's very straight forward in the documentation about what to do but i'm a bit lost with belongsToMany relationship. is there a practical example for this avaiable?

**Comment by ManojKiranA** [comment]
you can use relation manager
